hermes/send-email.js

```diff
@@ -5,7 +5,7 @@ const stringify = require('json-stringify-pretty-compact');
 import { deactiveUserEmailNotifications } from './models/usersSettings';
 
 let client;
-if (process.env.POSTMARK_SERVER_KEY) {
+if (process.env.POSTMARK_SERVER_KEY !== 'undefined') {
```

Comment:
`process.env.POSTMARK_SERVER_KEY` is `<string>undefined` in dev mode.
Did not look into the reason. This might also concern other parts of the application?

Reply:
That's intentional - we don't want to send any real emails with local data. To handle this we'll likely need to hand off the branch when you're done and I can do any final template testing with our postmark key :)

Reply:
maybe you misunderstood me, or i misunderstand you.

running in dev mode `process.env.POSTMARK_SERVER_KEY === 'undefined'` is true.
which means in dev mode we actually pass into the if statement, which i think is wrong.